internal/serverinstall/k8s.go

```diff
@@ -1353,6 +1484,28 @@ func (i *K8sInstaller) InstallFlags(set *flag.Set) {
 		Default: defaultServerImage,
 	})
 
+	set.StringVar(&flag.StringVar{
+		Name:    "k8s-odr-image",
+		Target:  &i.config.odrImage,
+		Usage:   "Docker image for the Waypoint On-Demand Runners",
+		Default: defaultODRImage,
+	})
+
+	set.StringVar(&flag.StringVar{
+		Name:   "k8s-runner-service-account",
+		Target: &i.config.odrServiceAccount,
+		Usage: "Service account to assign to the on-demand runner. If this is blank, " +
+			"a service account will be created automatically with the correct permissions.",
+		Default: "waypoint-runner",
+	})
+
+	set.BoolVar(&flag.BoolVar{
+		Name:    "k8s-runner-service-account-init",
+		Target:  &i.config.odrServiceAccountInit,
+		Usage:   "Create the service account if it does not exist.",
+		Default: true,
+	})
+
```

Comment:
Probably need this in `UpgradeFlags` as well?

Reply:
Hm! I didn't think about upgrades at all. Yes. Let me take a swing at that.